README.md

```diff
@@ -24,4 +24,8 @@ This service works great with [Rollout-Dashboard](https://github.com/fiverr/roll
 Edit `./config/redis.yml`
 
 ## How to start the service? 
-run `bundle exec rackup -p :port`
+Run 
+```
+ALLOWED_EMAILS="user1@gmail.com,user2@gmail.com" bundle exec rackup -p 4000
+```
+Specify proper ALLOWED_EMAILS and service port. In this example the service will listen on port 4000.
```

Comment:
1- please add a section in the FAQ that you can restrict access to specific users
2- maybe we should authenticate all calls to the API? 
3- since you're adding it to a READ operation, edit the README file

Reply:
2 - in scope of this PR I only want to make sure the main screen of Dashboard (i.e. the feature list) is secured.